projects/vdk-plugins/vdk-local-differential-privacy/README.md

```diff
@@ -0,0 +1,37 @@
+# VDK local differential privacy plugin
+
+This plugin adds support to VDK for anonymizing data before sending it for ingestion. 
+
+In most differential privacy algorithm we assume that the data curator(database owner) can be trusted. 
+The actual real data is stored in the database.
+The analyst can't be trusted and so noise is added to the result of the database queries.
+
+However this plugin should be used when the database curator is not trusted. 
+In this case we want to apply noise to the database before sending it to the curator.
+
+At the moment we support noise on two different types of data. 
+1. [Boolean data with Random response](https://programming-dp.com/ch13.html#randomized-response)
+2. [Enum type data with Unary encoding](https://programming-dp.com/ch13.html#unary-encoding)
+
+
+## Usage
+
+```
+pip install local-differential-privacy
```

Comment:
Reserve the name in pypi.org `local-differential-privacy` If you are providing instructions on people to install something. You need to make sure that we cannot suffer dependency confusion attack (https://fossa.com/blog/dependency-confusion-understanding-preventing-attacks/) 

I had to modify a few examples recently for that.